Dockerfile

```diff
@@ -1,3 +1,21 @@
+FROM swift:5.6-focal
+
+COPY . carton/
```

Comment:
Could you move this line after `apt-get install` to maximize cache opportunities?